packages/admin-stories/src/admin/toolbar/samples/TableForm.tsx

```diff
@@ -0,0 +1,152 @@
+                                }}
+                            >
+                                <Typography>Add</Typography>
+                            </Button>
+                        </>
+                    )}
+                >
+                    <ToolbarBackButton />
+                    <ToolbarTitleItem />
+                </Toolbar>
+                <MainContent>
+                    <Table
+                        data={data}
+                        totalCount={data.length}
+                        columns={[
+                            {
+                                name: "foo1",
+                                header: "Foo1",
+                            },
+                            {
+                                name: "foo2",
+                                header: "Foo2",
+                                render: (row) => <strong>{row.id}</strong>,
+                            },
+                            {
+                                name: "bar",
+                                visible: false,
+                            },
+                            {
+                                name: "nestedFoo.foo",
+                                header: "Nested foo",
+                            },
+                            {
+                                name: "actions",
+                                header: "",
+                                render: (row: IExampleRow) => (
+                                    <>
+                                        <IconButton
+                                            onClick={() => {
+                                                stackSwitchApi?.activatePage("edit", row.id.toString(10));
+                                            }}
+                                        >
+                                            <Edit color={"primary"} />
+                                        </IconButton>
+
+                                        <IconButton
+                                            onClick={() => {
+                                                onDeleteItem(row.id);
+                                            }}
+                                        >
+                                            <Delete color={"primary"} />
+                                        </IconButton>
+                                    </>
+                                ),
+                            },
+                        ]}
+                    />
+                </MainContent>
+            </>
+        );
+    };
+    const ExampleDetails = ({ mode, id }: { mode: "edit" | "create"; id: string }) => {
+        const item = data.find((element) => element.id.toString(10) === id);
+
+        return (
+            <>
+                <Toolbar>
+                    <ToolbarBackButton />
+                    <ToolbarTitleItem title={<Typography variant={"h4"}>{mode === "create" ? "Create" : `Edit Id: ${id}`}</Typography>} />
```

Comment:
I'd prefer 

```typescript
<ToolbarTitleItem>
     <Typography variant={"h4"}>{mode === "create" ? "Create" : `Edit Id: ${id}`}</Typography>
</ToolbarTitleItem>
```

Reply:
rendering title with children. i also added posibility to theme props that we can modify default typography props of title